.github/workflows/ruby.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on:
+  push:
+    branches:
+    - master
+
+pull_request:
+    branches:
+    - master
+    
+jobs:
+  test:
+    name: Test gem on Ruby ${{ matrix.ruby_version }} with Rails ${{ matrix.rails_version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby_version: [2.4 2.5 2.6]
+        rails_version: [5.0 5.1 5.2 6.0]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Ruby ${{ ruby_version }}
+      uses: actions/setup-ruby@v1
```

Comment:
I would suggest to use `ruby/setup-ruby@v1` like rails do, because there is no need to separately install bundler.